src/Formatter/SymfonyConsoleTextFormatter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+declare(strict_types=1);
+
+namespace Roave\ApiCompare\Formatter;
+
+use Roave\ApiCompare\Change;
+use Roave\ApiCompare\Changes;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class SymfonyConsoleTextFormatter implements OutputFormatter
+{
+    /**
+     * @var OutputInterface
+     */
+    private $output;
+
+    public function __construct(OutputInterface $output)
+    {
+        $this->output = $output;
+    }
+
+    public function write(Changes $changes) : void
+    {
+        /** @var Change $change */
+        foreach ($changes as $change) {
+            $this->output->writeln((string)$change);
```

Comment:
Add some color, be hipster.

Reply:
Nice idea; I'll add an issue but not bothering here

Reply:
See #16